WordPress/src/main/java/org/wordpress/android/GCMRegistrationIntentService.java

```diff
@@ -63,4 +66,22 @@ public void sendRegistrationToken(String gcmToken) {
             AppLog.w(T.NOTIFS, "Empty GCM token, can't register the id on remote services");
         }
     }
+
+
+    public static String getCurrentToken() {
```

Comment:
Uhh sorry, forgot to remove that function. It was my first attempt retrieving the token without storing it in Prefs.